scripts/ci/jenkins_pipeline_csi

```diff
@@ -1,4 +1,12 @@
 pipeline {
+    environment {
+        registryCredential = 'csi_w3_user'
+        artifactory_path = 'build/reports/images_url'
+        registry_protocol = 'https'
```

Comment:
do we expect this value ever to change/be used in more than 1 place?
if not, consider using the value directly without an additional variable.

Reply:
I don't think that we will change it but maybe it would change.
I don't think that it's harmful to use this variable, but if you still think that you prefer to use it directly I will change it.

Reply:
I still think I prefer to use it directly

Reply:
done